magento-engcom/php-7.2-support
# Update PHP version dependency to ~7.2.0

### Description
Add PHP 7.2 to `require-dev` of all `composer.json` files. Installation with composer will still fail due to third-party extension requirements and the `ext-mcrypt` requirement. 

### Fixed Issues (if relevant)
Fixes #15

### Manual testing scenarios
1. Checkout branch or clone repository
2. Run `composer install`
3. Installation should fail, but not due to Magento's PHP version constraints.

### Contribution checklist
